cypress/e2e/tests/administration/jira-connection/crud.test.ts

```diff
@@ -1,75 +1,71 @@
 import { login } from "../../../../utils/utils";
-import { getJiraConnectionData, getRandomCredentialsData } from "../../../../utils/data_utils";
-import { CredentialType, JiraType } from "../../../types/constants";
-import { CredentialsBasicJira } from "../../../models/administration/credentials/credentialsBasicJira";
-import { CredentialsData, JiraConnectionData } from "../../../types/types";
-import { CredentialsTokenJira } from "../../../models/administration/credentials/credentialsTokenJira";
+import { getJiraConnectionData, getJiraCredential } from "../../../../utils/data_utils";
+import { CredentialType } from "../../../types/constants";
+import { JiraCredentialsBasic } from "../../../models/administration/credentials/jiraCredentialsBasic";
+import { JiraConnectionData } from "../../../types/types";
+import { JiraCredentialsBearer } from "../../../models/administration/credentials/jiraCredentialsBearer";
 import { Jira } from "../../../models/administration/jira-connection/jira";
 
-describe(["@tier2"], "CRUD operations for Jira Server connection instance", () => {
+describe(["@tier2"], "CRUD operations for Jira Cloud instance", () => {
     const toBeCanceled = true;
-    const notToBeCanceled = false;
+    // const notToBeCanceled = false;
     const expectedToFail = true;
-    let validJiraBasicCredentials: CredentialsData;
-    let jiraBasicCredential: CredentialsBasicJira;
-    let jiraServerConnectionData: JiraConnectionData;
-    let jiraServerConnectionDataIncorrect: JiraConnectionData;
-    let jiraServerConnection: Jira;
-    const jira_url = Cypress.env("jira_url");
+    const useTestingAccount = true;
+    const useDummyAccount = false;
+    const isSecure = false;
+    let jiraBasicCredential: JiraCredentialsBasic | JiraCredentialsBearer;
```

Comment:
This is tied to my previous comment as to why  'JiraCredentialsBearer'  has been imported here.